### PR TITLE
Config file for sh1106 and st7789 (waveshare hat versions)

### PR DESCRIPTION
**Pull request by matemaciek**

### CODE REVIEW

**Comment by iceman1001** [comment]
```pi@raspberrypi:~/luma.examples/examples $ python3 bounce.py --display=st7789
Traceback (most recent call last):
  File "bounce.py", line 80, in <module>
    device = get_device()
  File "/home/pi/luma.examples/examples/demo_opts.py", line 63, in get_device
    device = cmdline.create_device(args)
  File "/usr/local/lib/python3.7/dist-packages/luma/core/cmdline.py", line 252, in create_device
    backlight_params = dict(gpio=interface.gpio, gpio_LIGHT=args.gpio_backlight, active_low=args.backlight_active == "low")
AttributeError: 'i2c' object has no attribute 'gpio'
```
When I try your conf,  luma.core seem to want to do a GPIO for lcd,

File: conf/sh1106.conf [hunk 1]
```diff
@@ -0,0 +1,2 @@
+--display=sh1106
+--interface=spi
```
Comment (thijstriemstra) [inline conf/sh1106.conf:2]: not sure about that one. can you rename it to ssh1106-waveshare.conf?

**Review by thijstriemstra — CHANGES_REQUESTED** [review]

**Comment by matemaciek** [comment]
> ```
> Traceback (most recent call last):
>   File "bounce.py", line 80, in <module>
>     device = get_device()
>   File "/home/pi/luma.examples/examples/demo_opts.py", line 63, in get_device
>     device = cmdline.create_device(args)
>   File "/usr/local/lib/python3.7/dist-packages/luma/core/cmdline.py", line 252, in create_device
>     backlight_params = dict(gpio=interface.gpio, gpio_LIGHT=args.gpio_backlight, active_low=args.backlight_active == "low")
> AttributeError: 'i2c' object has no attribute 'gpio'
> ```
> 
> When I try your conf, luma.core seem to want to do a GPIO for lcd,

Can you share command you're running?

**Comment by iceman1001** [comment]
```
python3 bounce.py -i spi -d st7789 --gpio-reset 27 --gpio-data-command 25 --gpio-backlight 24 --backlight-active high --spi-bus-speed 52000000
```

**Comment by iceman1001** [comment]
I had uninstall / install  core, oled, lcd,  examples,   and now it works :)

**Review by thijstriemstra — APPROVED** [review]

**Pull request merged**